modules/gitops-benefits-of-argo-rollouts.adoc

```diff
@@ -0,0 +1,26 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: CONCEPT
+[id="gitops-benefits-of-argo-rollouts_{context}"]
+= Benefits of Argo Rollouts
+
+While managing and coordinating advanced deployment strategies in traditional infrastructure involves long maintenance windows, automation with advanced infrastructure like {product-title} and {gitops-title} can help reduce these windows. However, setting up these strategies still requires a substantial effort. With Argo Rollouts, you can simplify this process with enhancements to enable application teams to define their rollout strategy declaratively. Teams no longer need to define multiple deployments and services or create automation for traffic shaping and integration of tests. Using Argo Rollouts, you can encapsulate all the required definitions of the declarative rollout strategy to automate and manage the process.
```

Comment:
This paragraph reminds me of marketing copy; I'm wondering if the "substantial effort" part casts our products in a negative light. Suggestion to consider:

"Managing and coordinating advanced deployment strategies in traditional infrastructure often involves long maintenance windows. Automation with tools like {product-title} and {gitops-title} can reduce these windows, but setting up these strategies can still be challenging. Argo Rollouts simplifies this process by allowing application teams to define their rollout strategy declaratively. This eliminates the need to define multiple deployments and services or create automation for traffic shaping and integration of tests. Argo Rollouts encapsulates all required definitions for a declarative rollout strategy, automating and managing the process."

Reply:
Thanks for the suggestions. I used most of them.